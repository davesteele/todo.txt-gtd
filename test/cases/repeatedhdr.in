# ProjA

Task A1

# ProjB

Task B1

# ProjA

Task A2

# _None

