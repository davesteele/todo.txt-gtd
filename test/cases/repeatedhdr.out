# ProjA

Task A1


Task A2

# ProjB

Task B1

# _None

